net-analyzer/ospd/ospd-1.3.2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+inherit eutils distutils-r1
+
+DESCRIPTION="Openvas OSP (Open Scanner Protocol)"
+HOMEPAGE="http://www.openvas.org/"
+SRC_URI="https://github.com/greenbone/ospd/archive/v1.3.2.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="extras"
+
+DEPEND="
+	dev-python/setuptools
+	>=dev-python/paramiko-2.4.2
+	dev-python/defusedxml
+	dev-python/lxml
+	>=net-analyzer/openvas-libraries-9.0.3
+"
+
+RDEPEND="${DEPEND}"
+
+BDEPEND="virtual/pkgconfig"
+
+DOCS=( CHANGES COPYING README.md )
+
+python_prepare_all() {
+	distutils-r1_python_prepare_all
+}
+
+python_compile() {
+	if use extras; then
+		/bin/bash "${S}"/doc/generate
```

Comment:
Will this work with prefix systems? I think you can safely just call bash, OR just execute the generate script itself. Also append with `|| die`.

Reply:
This still needs to be addressed.

Reply:
done now.